include/seqan3/argument_parser/validators.hpp

```diff
@@ -256,17 +257,33 @@ class file_ext_validator
 
     /*!\brief Constructing from a vector.
      * \param[in] v The vector of valid file extensions to test (e.g. {"fa", "fasta"}).
+     * \param[in] c Case sensitivity flag. Set true for case sensitivity. Default: false (case insensitive).
+     *              
+     * For case insensitivity, everything is converted to lower case characters.
      */
-    file_ext_validator(std::vector<std::string> const & v) :
-        extensions{v}
-    {}
+    file_ext_validator(std::vector<std::string> const & v, bool const c = false) :
+        case_sensitive{c}
+    {
+        if (c)
```

Comment:
I think we can even do that in one line :sunglasses: 
```suggestion
        extensions = (c) ? std::vector<std::string>{std::move(v)} : std::vector<std::string>{v | view::to_lower};
```
(then you need the cast to vector though to return the same type, but the move will prevent the copy)
if the line length is too long though I would just leave it as you have it now :)